src/test/kotlin/ratpack/example/kotlin/SiteSpec.kt

```diff
@@ -0,0 +1,72 @@
+package ratpack.example.kotlin
+
+import com.google.common.io.CharStreams
+import org.jetbrains.spek.api.Spek
+import ratpack.test.MainClassApplicationUnderTest
+import java.io.InputStreamReader
+import kotlin.test.assertEquals
+
+class SiteSpec : Spek({
+
+    val aut = MainClassApplicationUnderTest(Main::class.java)
```

Comment:
Great, thanks a lot!

I'll keep an eye on the open Ticket and if something like `@Closeable` becomes available, I'll update the implementation.